content/docs/other-guides/job-scheduling.md

```diff
@@ -4,18 +4,18 @@ description = "How to schedule a job with gang-scheduling"
 weight = 30
 +++
 
-This guide describes how to use kube-batch to support gang-scheduling in 
+This guide describes how to use volcano scheduler to support gang-scheduling in 
```

Comment:
Maybe add the volcano github link here instead of later, since readers may not be familiar with it.

Reply:
+1

Reply:
Added the volcano scheduler link